RecoTracker/IterativeTracking/python/InitialStepPreSplitting_cff.py

```diff
@@ -182,7 +182,12 @@
 )
 from Configuration.Eras.Modifier_pp_on_XeXe_2017_cff import pp_on_XeXe_2017
 from Configuration.ProcessModifiers.pp_on_AA_cff import pp_on_AA
-(pp_on_XeXe_2017 | pp_on_AA).toModify(firstStepPrimaryVerticesPreSplitting, TkFilterParameters = dict(trackQuality = 'any'))
+(pp_on_XeXe_2017 | pp_on_AA).toModify(firstStepPrimaryVerticesPreSplitting, 
+    TkFilterParameters = dict(
+        trackQuality = 'any',
+        maxNumTracksThreshold = 1000000
```

Comment:
I was wondering if the intention is to use a very large number,  could a python built-in constant be used instead?

Reply:
That is indeed the intention.  Could you provide a recommendation for a what an appropriate change would be?

Reply:
I was thinking about `sys.maxsize` (which kinda replaces `sys.maxint` in python3). Not sure if it is appropriate. Maybe also @cms-sw/reconstruction-l2 can comment here.

Reply:
Indeed, I would suggest to use `sys.maxsize`
```suggestion
        maxNumTracksThreshold = sys.maxsize
```

@abaty , could you please update it?

Reply:
@clacaputo 
This solution does not seem to work quite as intended.  Using `sys.maxsize ` returns the max for a 64-bit machine (9,223,372,036,854,775,807) which overflows the expected 32 bit int, causing an error.  Are you aware of an elegant way to return the max of a 32-bit int without hardcoding it in python 3?  Otherwise I could update the C++ to expect a 64 bit integer input.  Please advise which solution you would prefer.

Reply:
how about `2**31 -1`?